specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2021-12-01-preview/mysql.json

```diff
@@ -0,0 +1,2916 @@
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "IsConfigPendingRestart",
+            "modelAsString": true
+          }
+        },
+        "isDynamicConfig": {
+          "type": "string",
+          "readOnly": true,
+          "description": "If is the configuration dynamic.",
+          "enum": [
+            "True",
+            "False"
+          ],
+          "x-ms-enum": {
+            "name": "IsDynamicConfig",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The properties of a configuration."
+    },
+    "Configuration": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ConfigurationProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a configuration."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "The system metadata relating to this resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Represents a Configuration."
+    },
+    "ConfigurationListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Configuration"
+          },
+          "description": "The list of server configurations."
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      },
+      "description": "A list of server configurations."
+    },
+    "ConfigurationForBatchUpdate": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the configuration."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "type": "string",
+              "description": "Value of the configuration."
+            },
+            "source": {
+              "type": "string",
+              "description": "Source of the configuration."
+            }
+          },
+          "x-ms-client-flatten": true,
+          "description": "The properties can be updated for a configuration."
+        }
+      },
+      "description": "Represents a Configuration."
+    },
+    "ConfigurationListForBatchUpdate": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigurationForBatchUpdate"
+          },
+          "description": "The list of server configurations."
+        },
+        "resetAllToDefault": {
```

Comment:
>resetAllToDefault [](http://example.com/codeflow?start=9&length=17)

Consider using enum over boolean: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review)

Reply:
Fixed